### ISSUE TITLE
RTL support

### ISSUE THREAD

**Issue by ischas** [KlausBenndorf/guide4you#178]
The layerselector needs to be restructured to allow text to flow from right to left.

Other sections where text appears have to be checked.

**Comment by simonseyock** [comment]
The layerselector needs to be restructured to allow text to flow from right to left.

Other sections where text appears have to be checked.

**Comment by simonseyock** [comment]
I don't know if the layerselector really needs to be configured this way ... it would look a little bit strange or not?

**Comment by penpendede** [comment]
I disagree. It makes perfect sense if a Semite language is used. OpenStreetMap supports many languages and here is what you get if your browser's preferred language is set to Ivrit. Apart from the obvious differences, it looks the same for Arabic.

![image](https://cloud.githubusercontent.com/assets/9119675/20788141/be031c6e-b7af-11e6-815d-3750fd9d28cb.png)

However, if we are to RTL languages, the language switcher must be able to swap all element positions from left to right and vice versa. For comparison, here what you get German is preferred:

![image](https://cloud.githubusercontent.com/assets/9119675/20788394/e9b7651c-b7b0-11e6-8a89-b92b640276b2.png)
